help/develop/getting-started.md

```diff
@@ -3,208 +3,215 @@
 
 # Getting Started with Building on Datagrok
 
-Welcome to [Datagrok](https://www.datagrok.ai), the next generation data platform!
+Welcome to Datagrok, the next-generation data platform!
 
-This is a guide to help you jump-start using Datagrok as an engineer, extending Datagrok
-with plugins and building applications on top of it. During this Getting Started,
-post your questions and suggestions at our [Community Forum](https://community.datagrok.ai/).
-We usually respond within less than a day.
+This guide will help you jump-start using Datagrok as an engineer. In short, what you can do with Datagrok is extend it
+with plugins or build your applications on top of Datagrok.
+
+If you have any questions or suggestions after reading this Getting Started guide, go to our [Community Forum] &mdash; 
+typically respond within 24 hours.
 
 ## Let's start
 
-*Take it fast.* If you are already somewhat knowledgeable with the platform, maybe due to
-previous demos we had for you or having watched some of our
-[youtube](https://www.youtube.com/channel/UCXPHEjOd4gyZ6m6Ji-iOBYg) videos,
-you can take a fast track:   
-> Watch the [20 minutes to an app](https://www.youtube.com/watch?v=tVwpRB8fikQ) video refresher,
-> and proceed directly to the programming [Exercises](exercises.md).  
->  
-When you have time, check
-[Daily Source of Information](#daily-sources-of-information) to know where to post questions
-and get info on the platform updates. Return to this document time to time to study
-platform _in depth_.
-
-*Take it step by step.* In case you prefer a clear start and dive into the platform
-stepwise, we recommend you to follow the below *Table of Contents*:
+If you already know something about Datagrok (probably thanks to our [YouTube channel]), you can take a fast track:
+
+* Have a look at the [20-minute introduction to a Datagrok application] for a refresher, and then proceed to the
+  [JavaScript programming exercises](exercises.md) that explain how to use Datagrok with JavaScript.
```

Comment:
The phrase 'how to use Datagrok with JavaScript' sounds a bit unusual to me, could we replace it with 'how to work with Datagrok using JavaScript API' or something along those lines?

Reply:
Updated.